src/components/split-panel/index.js

```diff
@@ -0,0 +1,98 @@
+import { useState, useRef } from 'react';
+
+/**
+ * A design agnostic user resizable split panel controller.
+ * @param {function} children A `function(props)` for rendering the panels and handle (see implementation for props)
+ * @param {?number} [splitDefault=0.65] A number [0...1] as the default % split position
+ * @param {?number} [splitMin=0] A number [0...1] as the minimum % split position
+ * @param {?number} [splitMax=1] A number [0...1] as the maximum % split position
+ * @param {?number} [keyboardStep=0.025] A number [0...1] as the % step to move split when using keyboard
+ * @param {?string} [orientation='vertical'] Only 'vertical' currently supported.
+ * @return {object} The rendered children
```

Comment:
It would be more clear to add some example of what the children usually is and any expected format of the children in the comment, such as the react component it is encapsulating, etc

Reply:
Sure I've improved the documentation in this file, including a note that the test file serves as a reference usage.